fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingCommon.java

```diff
@@ -61,6 +61,7 @@
     @Nonnull private OnlineIndexer.IndexStatePrecondition indexStatePrecondition;
     @Nonnull public OnlineIndexer.Config config; // this item may be modified on the fly
     @Nullable private final Function<OnlineIndexer.Config, OnlineIndexer.Config> configLoader;
+    @Nullable private final Object scrubbingPolicy;
```

Comment:
Why is this an `Object`? Ideally, we wouldn't need it in the `IndexingCommon` at all as the scrubbers would be separate enough from the indexers that it wouldn't be something that would need to wind up all the way in `IndexingCommon` (maybe in `ScrubbingCommon` or something), but it seems weird that it's essentially untyped here

Reply:
The object, when non-null, returns true on `common.isScrubber()`. IndexingBase needs this information for the type-stamp. 
I thought it would be cleaner if the non-scrubbing modules wouldn't have to learn the ScrabbingPolicy's structure.

Reply:
Eliminated from common.